client/pages/recycle-and-reuse.js

```diff
@@ -24,25 +26,23 @@ const hasNoItems = (items) => items?.input?.length === 0;
 
 export async function getStaticProps() {
   //   const url = urlcat(process.env.SERVER_URL, "/api/items");
-  const url = urlcat("https://api.npoint.io", "a8416aa207861acb363d");
+  const url = urlcat("https://api.npoint.io", "324727202a8ce24af12f");
   try {
     const data = await axios.get(url);
     return {
       props: {
-        options: data.data,
+        data: data.data,
       },
     };
   } catch (error) {
-    return {
-      props: {
-        options,
-      },
-    };
+    // TODO: handle error in a better way
+    console.error(error)
```

Comment:
When there are errors returned by the API, we should deal with the error properly. But my update does not actually handle it well either 🌚